### PR TITLE
Move `accumulateListeners` to the only place where it's used.

### PR DESCRIPTION
**Pull request by cibernox**
I found that this private function was only used in one place within Ember and exactly 0 times in the wild, so I made it a local function in the file where it's used.

This avoids a few import/export for no reason. Also improved so we void peeking a
meta that we already have in the context.

### CODE REVIEW

**Comment by stefanpenner** [comment]
:shipit:

**Comment by cibernox** [comment]
@stefanpenner can you restart sauce?

**Comment by stefanpenner** [comment]
restarting sauce

**Comment by cibernox** [comment]
@stefanpenner again?

**Comment by stefanpenner** [comment]
restarting

**Comment by cibernox** [comment]
Is it normal that it fails so much or should I be worried?

**Comment by stefanpenner** [comment]
> Is it normal that it fails so much or should I be worried?

its been brittle for the last more or two, I haven't had a chance to investigate.

**Comment by stefanpenner** [comment]
@rwjblue should we just merge, code looks good...

**Comment by bekzod** [comment]
@stefanpenner as promised looked into IE instability, not sure what is the cause but to be safe just upgraded `web-driver` and `sauce-connect-launcher` in `saucie` (`ember-cli-sauce`) will push downstream once there is release in `saucie` https://github.com/johanneswuerbach/saucie/pull/71

File: packages/ember-metal/lib/property_events.js [hunk 1]
```diff
@@ -252,14 +251,49 @@ function changeProperties(callback, binding) {
...
+    if (target === array[i] && method === array[i + 1]) {
+      index = i;
+      break;
+    }
+  }
+  return index;
+}
+
+function accumulateListeners(obj, eventName, otherActions, meta) {
+  if (meta === null || meta === undefined) { return; }
```
Comment (bekzod) [inline packages/ember-metal/lib/property_events.js]: you can even remove meta existence check here since its always passed and its private api

Comment (cibernox) [inline packages/ember-metal/lib/property_events.js]: but it can be nil, can't it?

Comment (bekzod) [inline packages/ember-metal/lib/property_events.js]: No it can't, in particular use case of 'accummulateListeners' meta tha you are passing always not nil

Comment (cibernox) [inline packages/ember-metal/lib/property_events.js]: Updated.

**Comment by cibernox** [comment]
Oh, sauce is green!

**Comment by stefanpenner** [comment]
@cibernox merge merge merge!

**Pull request merged**